### DISCUSSION TITLE
Cubase Automation

### DISCUSSION THREAD

**Discussion by howidoo** [RomanKubiak/ctrlr#347] [Q&A]
I cant get cubase to read the automation. Im working on cubase 5 with ctrlr as VST. Writing is no problem when i turn the modulator in ctrlr cubase writes the automation line. But cntrlr dont read the automation. I have tried every plugin option and midi thru option but it doesnt work. Anyone has an indea? Thanks for helping.

**Comment by dnaldoog** [comment]
This is how I do it in Cubase 11. There might be a better way? But automation back to the panel always works for me.

1. In Cubase, Disable †Midi-in **Midi-out ports to the “Synth”.
2. Open  VST in an instrument track IN=(All Midi Inputs)
3. Create Midi track IN=(All Midi Inputs) OUT=(“Synth” Midi-in) (I record midi notes & data to this track).
4. In the VST Ctrlr panel connect to the †Midi-in and **Midi-out ~ connected to “Synth” (you disabled them in Cubase ( See step 1)).
5. I found the only Midi-thru setting that needs to be checked was MIDI→Midi Thru→”Plugin-host->Output Device” (The 5th option down in the fold out menu)
6. Engage Monitor on Both tracks.
